lib/client/lambda-server.ts

```diff
@@ -103,7 +103,9 @@ export class LambdaServer {
                 context.fail(Error(`LambdaServer input url should not contain more than '.' or node_modules.  found: ${onlyUrl}`));
                 return;
             }
-            [path, handlerFunction] = onlyUrl.split(".");
+            const splitUrl = onlyUrl.split(".");
```

Comment:
I think would be better to do array destructuring:

```typescript
const [path, handlerFunction] = onlyUrl.split(".");
```

Reply:
that seems to be broken builds for node 4 and node 5